plugin/js/EpubItemSupplier.js

```diff
@@ -18,12 +18,13 @@ EpubItemSupplier.prototype.setChapters = function (chapters) {
 }
 
 EpubItemSupplier.prototype.createXhtmlFileName = function (fileIndex) {
-    return "index_split_" + util.zeroPad(fileIndex) + ".html";
+    // Belldandu: For now only new format gets implemented because not sure where this files functions are even called and i dont want to accidentally break something.
+    return "OEBPS/Text/[" + util.zeroPad(fileIndex) + "]" + ".xhtml";
```

Comment:
Chapter Name was not done here because i'm not even sure this function gets called so there is no title to send to it.
Instead it follows part of the new format which is ```[0000].xhtml``` and so on.